src/wheatley.ts

```diff
@@ -720,6 +720,63 @@ export class Wheatley {
         }
     }
 
+    /**
+     * Search for a specific channel by name if the provided name is not found then instead search by id.
+     */
+    async get_channel(name: string, id: string, guild_to_check: Discord.Guild = this.TCCPP) {
+        const channel_by_name = guild_to_check.channels.cache.find(channel => channel.name === name);
+        if (channel_by_name) {
+            return channel_by_name;
+        }
+
+        const channel_by_id = guild_to_check.channels.cache.get(id);
+        if (channel_by_id) {
+            return channel_by_id as Discord.GuildChannel;
+        }
+
+        return null;
+    }
+
+    /**
+     * Search for a specific channel by name if the provided name is not found then instead
+     * search by id. If all attempts to find a channel fails then create a new channel.
+     * @remarks This function will act the same as get_channel when we are not in production.
+     * @remarks Currently fallback_channel_type will always create a text channel. This will change at a later date.
+     */
+    async get_channel_or_create_channel(
```

Comment:
I think this logic should be folded into `get_channel`, I don't think there's anywhere in the bot where calling `get_channel_or_create_channel` alone would make sense

Reply:
Honestly for the time being I'll just remove it. We can revisit it at a later date when boot strapping is actually happening.